nextflow_schema.json

```diff
@@ -251,6 +251,40 @@
                     "description": "Maximum window size before and after variant locii used to generate ASCIIGenome screenshots.",
                     "fa_icon": "fab fa-windows"
                 },
+                "skip_freyja": {
+                    "type": "boolean",
+                    "fa_icon": "fas fast-forward",
+                    "description": "Skip freyja deep SARS-CoV-2 variant analysis using a depth weighted approach."
+                },
+                "freyja_db_name": {
+                    "type": "string",
+                    "default": "freyja_db",
+                    "hidden": true,
+                    "fa_icon": "fas fa-folder-open",
+                    "description": "Specify the name where to store UShER database (default: 'freyja_db')."
+                },
+                "freyja_repeats": {
+                    "type": "integer",
+                    "default": 100,
+                    "fa_icon": "fas fa-hand-paper",
+                    "description": "Specify the number of bootstrap repeats to do."
+                },
+                "freyja_barcodes": {
+                    "type": "string",
+                    "format": "path",
+                    "hidden": true,
+                    "default": "[]",
```

Comment:
I don't think we should have `[]` as default

Reply:
You are tottaly right, I must confess that I edited this file before I knew the presence of `nf-core schema build` (the latest commit was done with it).

Reply:
I am fixing now.